etc/c/curl.d

```diff
@@ -879,8 +881,10 @@ enum CurlOption {
   /** The CApath or CAfile used to validate the peer certificate
      this option is used only if SSL_VERIFYPEER is true */
   cainfo = 10065,
-  /** 66 = OBSOLETE */
-  /** 67 = OBSOLETE */
+  /** $(UL */
```

Comment:
That surprised me because it *does* work for /// doc comments. Adjacent /// get merged before processing. At least it seems to work fine based on the results of https://github.com/D-Programming-Language/druntime/pull/1408 over at http://dtest.thecybershadow.net/artifact/website-fa397df5041e797d4ecdefd0231aa6ae97f9ea34-52af7a31c124ce2ec1c988269811b9fb/web/library-prerelease/core/cpuid/prefer_athlon.html